distributed/tests/test_actor.py

```diff
@@ -488,7 +488,7 @@ def check(counter, blanks):
     start = time()
     while a.data or b.data:
         await asyncio.sleep(0.01)
-        assert time() < start + 30
+        assert time() < start + 10, (dict(a.data), dict(b.data))
```

Comment:
This test randomly failed twice over 360 runs (the Actor was still in Worker.data after 10 seconds). Why this happened is out of scope here.

What is relevant to this PR, however, is that both times the flaky decorator reran the test 10 times, and it failed 10 times in a row. Why this happens completely baffles me. I've already tested that c, s, a, b are regenerated from scratch at every rerun.

Reply:
See log: https://github.com/dask/distributed/pull/5022/checks?check_run_id=3003465962

Reply:
Might be related to https://github.com/dask/distributed/pull/4937